packages/sdks-tests/src/tests/basic-styles.spec.ts

```diff
@@ -0,0 +1,83 @@
+import { expect } from '@playwright/test';
+import { test, __dirname, isRNSDK } from './helpers/index.js';
+import fs from 'fs';
+import path from 'path';
+
+test.describe('Basic styles', () => {
+  test.fail(isRNSDK);
+  test('styles should be applied consistently across all SDKs (text and buttons)', async ({
+    page,
+    packageName,
+  }) => {
+    test.skip(packageName === 'react-native');
+    await page.goto('/basic-styles');
+
+    const button = page.getByRole('button', { name: 'Click Me!' });
+
+    const buttonWidth = await button.evaluate(el => el.getBoundingClientRect().width);
+    expect(buttonWidth).toBe(1264);
+
+    const centeredAlignedText = page.locator('text=Centered aligned');
+    const leftAlignedText = page.locator('text=Left aligned');
+    const rightAlignedText = page.locator('text=Right aligned');
+
+    expect(await centeredAlignedText.evaluate(el => getComputedStyle(el).color)).toBe(
+      'rgb(0, 255, 16)'
+    );
+    expect(
+      await centeredAlignedText.evaluate(el => el.getBoundingClientRect().left)
+    ).toBeGreaterThan(580);
+
+    expect(await leftAlignedText.evaluate(el => getComputedStyle(el).color)).toBe('rgb(255, 0, 0)');
+    expect(await leftAlignedText.evaluate(el => el.getBoundingClientRect().left)).toBeLessThan(10);
+
+    expect(await rightAlignedText.evaluate(el => getComputedStyle(el).color)).toBe(
+      'rgb(0, 24, 255)'
+    );
+    expect(await rightAlignedText.evaluate(el => el.getBoundingClientRect().right)).toBeGreaterThan(
+      1200
+    );
```

Comment:
Why not `expect(locator).toHaveCSS(key, value)`?

Reply:
the computed value says `0px` margin-left/right so we cannot fully predict if its actually working) -- the alignment